[2019-01-09 17:27:22,994] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Too many connections
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-09 18:03:46,295] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Too many connections
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-09 18:09:41,024] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Too many connections
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-09 18:17:47,965] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Too many connections
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-09 18:25:04,398] WARN  {org.wso2.transport.http.netty.contractimpl.sender.TargetHandler} - Exception occurred in TargetHandler : Connection reset by peer  
[2019-01-09 18:25:04,411] ERROR {org.wso2.transport.http.netty.contractimpl.sender.states.RequestCompleted} - Remote host closed the connection before initiating inbound response  
[2019-01-09 18:34:44,851] WARN  {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=1h36m28s540ms).  
[2019-01-10 10:18:07,481] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-10 10:19:07,254] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-10 10:21:34,433] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-10 10:25:12,277] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-10 10:26:42,090] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-10 10:31:45,288] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
[2019-01-10 10:34:43,628] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLException: Access denied for user ''@'localhost' (using password: NO)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:835)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createServerBasedDBClient(SQLDatasourceUtils.java:1156)
	at org.ballerinalang.database.mysql.CreateClient.execute(CreateClient.java:61)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:920)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:882)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:434)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:184)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:267)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:171)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:163)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:302)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
 
